### ISSUE TITLE
BBC: Blead breaks MLEHMANN/Coro-6.57.tar.gz

### ISSUE THREAD

**Issue by eserte** [Perl/perl5#21784]
If EV or Event are installed, then the test suite fails since perl 5.39.3 like this (in the Event subdirectory):
```
...
t/01_unblock.t .. 1/12 Attempt to free unreferenced scalar: SV 0x1102708 at /home/cpansand/.cpan/build/2023123113/Coro-6.57-0/Event/../blib/lib/Coro.pm line 442.
t/01_unblock.t .. Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 2/12 subtests 
...
```
or like this (in the EV subdirectory):
```
t/01_unblock.t .. 1/12 Attempt to free unreferenced scalar: SV 0x1e45f30 at /home/cpansand/.cpan/build/2023123113/Coro-6.57-0/EV/../blib/lib/Coro.pm line 442.
t/01_unblock.t .. Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 2/12 subtests 
```

CPAN Testers <http://matrix.cpantesters.org/?dist=Coro+6.57> show also pass reports. In these cases probably neither Event nor EV were installed.

**Comment by jkeenan** [comment]
@eserte, it's not clear to me how you got these results.  On Linux, I twice attempted to bisect this problem, using these invocations:
```
 2095  perl Porting/bisect.pl --start=v5.39.2 --end=v5.39.3 --with-module=Event --module AnyEvent
 2097  perl Porting/bisect.pl --start=v5.39.2 --end=v5.39.3 --with-module=EV --module AnyEvent
```
In both cases, no failure was reported:
```
Runner returned 0 for end revision at Porting/bisect.pl line 233.
```
Could you clarify?  Thanks.

**Comment by jkeenan** [comment]
> @eserte, it's not clear to me how you got these results. On Linux, I twice attempted to bisect this problem, using these invocations:
> 
> ```
>  2095  perl Porting/bisect.pl --start=v5.39.2 --end=v5.39.3 --with-module=Event --module AnyEvent
>  2097  perl Porting/bisect.pl --start=v5.39.2 --end=v5.39.3 --with-module=EV --module AnyEvent
> ```
> 
> In both cases, no failure was reported:
> 
> ```
> Runner returned 0 for end revision at Porting/bisect.pl line 233.
> ```
> 
> Could you clarify? Thanks.

@eserte, could you clarify how you got these BBCs?  Thanks.

**Comment by guest20** [comment]
Coro isn't in AnyEvent... Could that be it?

**Comment by haarg** [comment]
This bisects to e594d9080c26a5c6a42d9b9cf367bff5b1acfba2

**Comment by iabyn** [comment]
Should be fixed in blead now by v5.39.8-73-g0476589cca
